common/app/views/fragments/photoEssay.scala.html

```diff
@@ -8,15 +8,17 @@
 @defining((
     page.item.elements.hasMainEmbed,
     page.item.fields.main.nonEmpty,
-    page.item.elements.hasMainVideo
-)) { case (hasEmbed, hasMainMedia, hasVideo) =>
+    page.item.elements.hasMainVideo,
+    page.item.tags.isPaidContent
+)) { case (hasEmbed, hasMainMedia, hasVideo, isPaidContent) =>
   <style>
     @Html(common.Assets.css.inlinePhotoEssay)
   </style>
 
     <div class="content--photo-essay @RenderClasses(Map(
             "immersive-main-media" -> hasMainMedia,
-            "photo-essay--video" -> hasVideo
+            "photo-essay--video" -> hasVideo,
+            "immersive-main-media__advertisment" -> isPaidContent
```

Comment:
Eagle eyes! Hah, that class is only used in two places, the outmost \<DIV\> of the templates in`immersiveMainMedia.scal.html` and now `photoEssay.scala.html`. 

This class isn't currently linked to any CSS... or doing anything at all. Neither is the correct spelling of `immersive-main-media__advertisement`. 😅 Which leads me to the question, do I fix the spelling and keep applying the classes? or do I just delete this line and line 23 of `immersiveMainMedia.scal.html`?